README.md

```diff
@@ -8,9 +8,12 @@ The jsonnet-bundler is a package manager for [Jsonnet](http://jsonnet.org/).
 ## Install
 
 ```
-go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+GO111MODULE="on" go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
 ```
+**NOTE**: please use the latest go to do this, ideally go 1.13 or greater.
 
+This will put `jb` in `$(go env GOPATH)/bin`. If you encounter the error
+`jb: command not found` after installation then you may need to either add that directory to your `$PATH` as shown [here](https://golang.org/doc/code.html#GOPATH)
```

Comment:
```suggestion
`jb: command not found` after installation then you may need to add the Go binary directory to your `$PATH` as shown [in their docs](https://golang.org/doc/code.html#GOPATH).
```

Using "here" in links is hard for screen-readers, using a descriptive name is better :D
Furthermore, looks like the `either` is misplaced there :)